app/resources/css-default.template

```diff
@@ -194,6 +196,29 @@ input {
   font-size: 13px;
   margin-bottom: .5em;
   max-width: 100%;
+  border: 0;
```

Comment:
Currently you apply this for all "input" and then handle input[type="file"] as an exception.

How about instead apply this for limited types of inputs where you want to apply this style? Here are list of possible types:
https://www.w3.org/TR/REC-html40/interact/forms.html#adef-type-INPUT
and I believe there are only a few of them (text, password?) where this styling is relevant.